### PR TITLE
Add new sensors to airq integration

### PR DESCRIPTION
**Pull request by Sibgatulin**
## Proposed change

Recently a bunch of new sensors has been added to the device, so the integration is being updated.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/90413
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by emontnemery — COMMENTED** [review]
According to the core PR, new sensors are added in new firmware versions. Should the table include a minimum firmware version for the new sensors?

**Comment by Sibgatulin** [comment]
Good point @emontnemery. Frankly, I am not completely sure, as the devices which have these physical sensors already have the necessary firmware version, while for older devices a firmware update will not change anything in the operation of the integration. 

_Except_ for a few of the new virtual sensors, namely "Relative Pressure" and "Virus Index", which need to be explicitly activated. This, however, cannot be done by the integration in its present state and the users need to follow instructions from the devices non-public documentation. 

Would the following be a suitable addition to this PR or should I open another?
```
Virtual sensors "Relative Pressure" and "Virus Index" are introduced in firmware v1.80.0 but deactivated by default. Instructions to activate can be found in the [technical documentation](https://docs.air-q.com/).
```
(The core PR mentions firmware v1.82.0, which introduced for the bulk of the new sensors, but not this two special cases.)

File: source/_integrations/airq.markdown [hunk 1]
```diff
@@ -25,43 +25,64 @@ During the configuration, the user is prompted for the IP address of the device
...
+| Virus Index          | %                   |
 | VOC                  | ppb                 |
 | VOC (Industrial)     | ppb                 |
 
 PM1, PM25, and PM10 correspond to concentrations of particulates with diameter less than 1µm, 2.5µm, and 10µm respectively
 
-After the setup, separate entities will be created for each of the aforementioned sensors, found in your device.
+Virus Index uses CO2 as a proxy for potential aerosol load and can be seen as an indicator of ventilation sufficiency (0 %: insufficient ventilation, 100 %: all fine).
+
+Virtual sensors "Relative Pressure" and "Virus Index" are introduced in firmware v1.80.0 but deactivated by default. Users can activate them in the air-Q mobile application under "Advanced settings".
```
Comment (frenck) [inline source/_integrations/airq.markdown:88]:
```suggestion
Virus Index uses CO2 as a proxy for potential aerosol load and can be seen as an indicator of ventilation sufficiency (0 %: insufficient ventilation, 100 %: all fine).

Virtual sensors "Relative Pressure" and "Virus Index" are introduced in firmware v1.80.0 but deactivated by default. You can activate them in the air-Q mobile application under "Advanced settings".
```

Comment (frenck) [inline source/_integrations/airq.markdown]:
```suggestion
Virtual sensors "Relative Pressure" and "Virus Index" are introduced in firmware v1.80.0 but deactivated by default. You can activate them in the air-Q mobile application under "Advanced settings".
```

**Review by frenck — APPROVED** [review]
Thanks, @Sibgatulin 👍 

../Frenck

**Pull request merged**